packages/utils/src/useColumnManager/helper.js

```diff
@@ -0,0 +1,8 @@
+import get from 'lodash/get';
+
+export const filterColumnsBySelected = (columns, selected, selectProp) => columns.filter((column) => selected.includes(get(column, selectProp)));
```

Comment:
We should use default values for helper functions, you never know who's going to use them
```suggestion
export const filterColumnsBySelected = (columns = [], selected = false, selectProp = '') => columns.filter((column) => selected.includes(get(column, selectProp)));
```